### DISCUSSION TITLE
what does "label" of atomic_dipole.raw refer to?

### DISCUSSION THREAD

**Discussion by singlebook** [deepmodeling/deepmd-kit#719] [Q&A]
Is it the dipole of each atom calculated by AIMD?  DP tutorial did not make it clear.

**Comment by YWolfeee** [comment]
`atomic_dipole` can be any 3-dimensional vector defined in **atom**, and the dipole calculated by AIMD is one of that. By contrast, `global_dipole`  is the corresponding vector defined in the **system**. 

For example, in `examples/water_tensor/dipole` (`devel` branch), `atomic_dipole` is a matrix with shape `nframes * (n * 3)`, where `n` is the number of atom `O`.
